cms/djangoapps/contentstore/features/component.py

```diff
@@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
                 'No problem was added to the unit.')
 
     problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
-    actual_text = world.css_text(problem_css)
-    assert_in(category.upper(), actual_text)
+    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
```

Comment:
It was an interesting one. This: https://bitbucket.org/logilab/pylint/issue/457/spurious-no-value-for-parameter-error-on